insights/specs/default.py

```diff
@@ -223,6 +224,23 @@ def is_ceph_monitor(broker):
     cinder_api_log = first_file(["/var/log/containers/cinder/cinder-api.log", "/var/log/cinder/cinder-api.log"])
     cinder_conf = first_file(["/var/lib/config-data/puppet-generated/cinder/etc/cinder/cinder.conf", "/etc/cinder/cinder.conf"])
     cinder_volume_log = first_file(["/var/log/containers/cinder/volume.log", "/var/log/containers/cinder/cinder-volume.log", "/var/log/cinder/volume.log"])
+
+    @datasource(HostContext)
+    def cloud_init_network(broker):
+        relative_path = '/etc/cloud/cloud.cfg'
+        network_config = ''
+
+        with open(relative_path, 'r') as f:
```

Comment:
Exception will be thrown when the file does not exist, how about skipping the `open` gracefully instead of throwing exception when file doesn't exist?

Reply:
Fixed.

```
$ insights-cat --no-header cloud_init_network
Missing Dependencies:
    At Least One Of:
        insights.specs.default.DefaultSpecs.cloud_init_network
        insights.specs.insights_archive.InsightsArchiveSpecs.cloud_init_network

```